docs/user/filesystem.md

```diff
@@ -218,6 +218,265 @@ dfuse           9.4G  326K  9.4G   1% /tmp/dfuse
 DFuse can be launched via fstab and the standard mount command, it will parse -o options
 and extract pool=<name>,container=<name> if provided and ignore any other filesystem options given.
 
+There are few use cases described below to explain how systemd or /etc/fstab can be used to mount
+the daos container using dfuse.
+
+#### 1> User can mount the file system using mount.fuse3 command
+
+```bash
+$ dmg pool create --scm-size=8G --nvme-size=64G --label=samirrav_pool -u samirrav@
+Creating DAOS pool with manual per-engine storage allocation: 8.0 GB SCM, 64 GB NVMe (12.50% ratio)
+Pool created with 11.11%,88.89% storage tier ratio
+--------------------------------------------------
+  UUID                 : 5d27e7b1-a4de-4cae-a2e9-92b8bad11347
+  Service Ranks        : [1-5]
+  Storage Ranks        : [0-7]
+  Total Size           : 576 GB
+  Storage tier 0 (SCM) : 64 GB (8.0 GB / rank)
+  Storage tier 1 (NVMe): 512 GB (64 GB / rank)
+
+$ daos cont create samirrav_pool samirrav_cont --type=POSIX
+  Container UUID : 5091a6d5-f441-4ccc-8acb-5480ae879b33
+  Container Label: samirrav_cont
+  Container Type : POSIX
+
+Successfully created container 5091a6d5-f441-4ccc-8acb-5480ae879b33
+
+$ daos cont get-prop samirrav_pool samirrav_cont
+Properties for container samirrav_cont
+Name                                    Value
+----                                    -----
+Highest Allocated OID                   0
+Checksum                                off
+Checksum Chunk Size                     32 KiB
+Compression                             off
+Deduplication                           off
+Dedupe Threshold                        4.0 KiB
+EC Cell Size                            64 KiB
+Performance domain affinity level of EC 1
+Encryption                              off
+Global Version                          2
+Group                                   samirrav@
+Label                                   samirrav_cont
+Layout Type                             POSIX (1)
+Layout Version                          1
+Max Snapshot                            0
+Owner                                   samirrav@
+Redundancy Factor                       rf0
+Redundancy Level                        node (2)
+Performance domain affinity level of RP 3
+Server Checksumming                     off
+Health                                  HEALTHY
+Access Control List                     A::OWNER@:rwdtTaAo, A:G:GROUP@:rwtT
+
+$ mount.fuse3 dfuse /tmp/daos_dfuse_samir -o pool=samirrav_pool,container=samirrav_cont
```

Comment:
I know you can do this but I don't think there's any expectation that any user ever would call any of the mount.* binaries directly, for this example I'd stick to the regular was of dfuse mounting.